### PR TITLE
Refs task #401133 bug #401379 Replace token by url, error 404 when click payload history log

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#401133](https://dev.sun-asterisk.com/issues/401133)
- [#401379](https://dev.sun-asterisk.com/issues/401379)

## What's this PR do ?

- [x] Replace token by url in webhook show/update screen
- [x] Fix Value of column log in list payload history redirect to page not found

## Library
*(List package, library third party add new include version)*

- N/A

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*

image

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr></table>

**Comment by ghost** [comment]
ready

File: resources/views/admins/payload_histories/index.blade.php [hunk 1]
```diff
@@ -79,10 +79,10 @@
                                 <div class="label label-danger">Failed</div>
                             @endif
                         </td>
-                        <td><a href="page_ready_user_profile.html">{{ $payloadHistory->log }}</a></td>
+                        <td><a href="{{ route('history.show', ['history' => $payloadHistory->id]) }}">{{ $payloadHistory->log }}</a></td>
```
Comment (quanghung97) [inline resources/views/admins/payload_histories/index.blade.php]: hình như cái này là trang admin. thì route của nó là admin.history.show

Comment (ghost) [reply resources/views/admins/payload_histories/index.blade.php]: sorry chú , nãy cop bên user quên edit mất

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr></table>

**Review by quanghung97 — APPROVED** [review]
OK

**Review by nguyenthinhhn — APPROVED** [review]

**Review by tamtt-1031 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**